src/http/server/response.cr

```diff
@@ -154,6 +159,7 @@ class HTTP::Server
         @sync = false
         @flush_on_newline = false
         @chunked = false
+        @closed = false
```

Comment:
@asterite This is in `#reset`. I'm not sure how useful this is, because when a request is closed, at least the headers has already been written to the underlying IO. This effectively can't be reset because the response is already sent.

Reply:
Oh, cool. Reset is useful, can't remember now why, but response is reused for keep alive, check request processor.

Reply:
Oh yeah, `RequestProcessor` resets the response to avoid allocating a new instance for each iteration.

Reply:
I'll add a comment about this.